elide-core/src/test/java/com/yahoo/elide/audit/LogMessageTest.java

```diff
@@ -69,30 +71,34 @@ public void verifyListExpressions() {
         final String[] expressionForDefault = { "${child.id}" };
         final LogMessage message = new LogMessage("{0} {1} {2}", friendRecord, expressions, 1, Optional.empty());
         final LogMessage defaultMessage = new LogMessage("{0}", friendRecord, expressionForDefault, 1, Optional.empty());
-        Assert.assertEquals(message.getMessage(), "5 9 7", "JEXL substitution evaluates correctly.");
-        Assert.assertEquals(defaultMessage.getMessage(), "9", "JEXL substitution evaluates correctly.");
-        Assert.assertEquals(message.getChangeSpec(), Optional.empty());
+        assertEquals("5 9 7", message.getMessage(), "JEXL substitution evaluates correctly.");
+        assertEquals("9", defaultMessage.getMessage(), "JEXL substitution evaluates correctly.");
+        assertEquals(Optional.empty(), message.getChangeSpec());
     }
 
 
-    @Test(expectedExceptions = InvalidSyntaxException.class)
+    @Test
     public void invalidExpression() {
         final String[] expressions = { "${child.id}, ${%%%}" };
-        new LogMessage("{0} {1}", childRecord, expressions, 1, Optional.empty()).getMessage();
+        assertThrows(
+                InvalidSyntaxException.class,
+                () -> new LogMessage("{0} {1}", childRecord, expressions, 1, Optional.empty()).getMessage());
     }
 
-    @Test(expectedExceptions = InvalidSyntaxException.class)
+    @Test
     public void invalidTemplate() {
         final String[] expressions = { "${child.id}" };
-        new LogMessage("{}", childRecord, expressions, 1, Optional.empty()).getMessage();
+        assertThrows(
+                InvalidSyntaxException.class,
+                () -> new LogMessage("{}", childRecord, expressions, 1, Optional.empty()).getMessage());
     }
 
     public static class TestLoggerException extends RuntimeException {
     }
 
     private AuditLogger testAuditLogger = new Slf4jLogger();
 
-    @Test(threadPoolSize = 10, invocationCount = 10)
+    @Test
```

Comment:
The point of this tests was to test thread safety.  This is not an equivalent test.